shell/platform/linux/fl_platform_plugin.cc

```diff
@@ -335,6 +337,11 @@ static void method_call_cb(FlMethodChannel* channel,
     response = system_sound_play(self, args);
   } else if (strcmp(method, kSystemNavigatorPopMethod) == 0) {
     response = system_navigator_pop(self);
+  } else if (strcmp(method, kInitializationCompleteMethod) == 0) {
+    // TODO(gspencergoog): Handle this message to enable exit message listening.
+    // https://github.com/flutter/flutter/issues/126033
+    response =
+        FL_METHOD_RESPONSE(fl_method_success_response_new(fl_value_new_null()));
```

Comment:
This leaks the value from `fl_value_new_null` - this can be replaced with `fl_method_success_response_new(nullptr))` as used elsewhere in the code.

Reply:
Thank you, please check again. Out of curiosity, when would calling `fl_value_new_null` actually be desired?

Reply:
Using `nullptr` in this case is a shortcut as the `fl_value_new_null()` is then done within `fl_method_success_response_new`. You're unlikely to ever need to actually make a null object, it's really only useful on received messages.